packages/flutter/lib/src/painting/text_span.dart

```diff
@@ -338,7 +305,15 @@ class TextSpan extends InlineSpan {
     return null;
   }
 
-  @override
+  /// Populates the `semanticsOffsets` and `semanticsElements` with the appropriate data
+  /// to be able to construct a [SemanticsNode].
+  ///
+  /// If applicable, the beginning and end text offset are added to [semanticsOffsets].
+  /// [PlaceholderSpan]s have a text length of 1, which corresponds to the object
+  /// replacement character (0xFFFC) that is inserted to represent it.
+  ///
+  /// Any [GestureRecognizer]s are added to `semanticsElements`. Null is added to
+  /// `semanticsElements` for [PlaceholderSpan]s.
```

Comment:
Is the `@override` no longer needed here? or is this an oversight?

Reply:
No longer needed. :)